misc/nvram.md

```diff
@@ -36,57 +34,57 @@ If folders are empty then delete them as well:
 
 ## Verifying if you have working NVRAM
 
-To start, open the terminal and run the following one line at a time:
+To start, open the terminal and run the following command, which sets a variable named `test` in your NVRAM to the current date and time:
 
 ```sh
-sudo -s
-nvram -c
-nvram myvar=test
-exit
+sudo nvram myvar="$(date)"
 ```
 
 Now reboot and run this:
 
 ```sh
-nvram -p | grep -i myvar
+nvram myvar
 ```
 
-If nothing returns then your NVRAM is not working. If a line containing `myvar test` returns, your NVRAM is working.
-
-Note: `nvram -c` requires SIP to be off, an alternative is to wipe NVRAM at the boot menu. Reminder you'll need `Misc -> Security -> AllowNvramReset -> YES`
+If nothing returns then your NVRAM is not working. If a line containing `myvar` and then the current date, your NVRAM is working.
 
-## Enabling emulated NVRAM (with a nvram.plist)
+## Emulating NVRAM (with a `nvram.plist`)
 
-To enable emulated NVRAM, you'll need 3 things set:
+If you don't have native NVRAM, don't fret. We can set up emulated NVRAM by using a script to save the NVRAM contents to a plist during the shutdown process, which will then be loaded by OpenCore at the next startup.
 
-![](../images/post-install/nvram-md/nvram.png)
+To enable emulated NVRAM, you'll need the following set:
 
 Within your config.plist:
 
-* **Booter**:
+* **Booter -> Quirks**:
   * `DisableVariableWrite`: set to `NO`
 * **Misc -> Security**:
-  * `ExposeSensitiveData`: set to `0x3`
+  * `ExposeSensitiveData`: set to at least `0x1`
 * **NVRAM**:
-  * `LegacyEnable`: set to `YES`
   * `LegacyOverwrite` set to `YES`
-  * `LegacySchema`: NVRAM variables set(OpenCore compares these to the variables present in nvram.plist)
+  * `LegacySchema`: NVRAM variables set (OpenCore compares these to the variables present in `nvram.plist`)
   * `WriteFlash`: set to `YES`
 
 And within your EFI:
 
-* `OpenRuntime.efi` driver(this is needed for proper sleep, shutdown and other services to work correctly
+* `OpenVariableRuntimeDxe.efi` driver (with `LoadEarly` set to `YES`, and placed _before_ OpenRuntime)
```

Comment:
Maybe specify that the order and LoadEarly need to be set in the Config.plist. Order of files within the directory doesn't really matter.